apps/atr-one/config/system/boot/centrifugo.rb

```diff
@@ -0,0 +1,13 @@
+# Be sure to name this file after the component
+
+AtrOne::Container.boot(:centrifugo) do |container|
+  init do
+    secrets = JSON.parse(File.read('/run/secrets/centrifugo.json'))
+
+    client = Cent::Client.new(api_key: secrets['api_key'], endpoint: ENV['CENTRIFUGO_ENDPOINT'])
```

Comment:
Note how I have removed the endpoint from secrets. It's not a secret, it's a configurable setting.